sdk/recoveryservices-backup/Microsoft.Azure.Management.RecoveryServices.Backup/src/Microsoft.Azure.Management.RecoveryServices.Backup.csproj

```diff
@@ -12,6 +12,10 @@
 		<PackageReleaseNotes>
 			<![CDATA[
 New features:
+- Added changes for Azure Backup Health.
+       ]]>
+			<![CDATA[
+New features:
 - Added 4 new rest api related to private endpoint connection work (JEDI).
        ]]>
 		</PackageReleaseNotes>
```

Comment:
@lapradh this looks like a bad merge. There should be one `<![CDATA[` `]]>` block within `<PackageReleaseNotes>`.

Reply:
Pls review now